eland/field_mappings.py

```diff
@@ -584,21 +584,19 @@ def aggregatable_field_name(self, display_name: str) -> Optional[str]:
 
         raise KeyError if the field_name doesn't exist in the mapping, or isn't aggregatable
         """
-        if display_name not in self._mappings_capabilities.index:
+        mapping: Optional[pd.Series] = None
+
+        try:
+            mapping = self._mappings_capabilities.loc[display_name]
+        except KeyError:
             raise KeyError(
```

Comment:
Let's add `from None` to this `raise` so our traceback doesn't change ie: `raise KeyError("...long message") from None`